app/components/mods_display/row_field_component.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module ModsDisplay
+  class RowFieldComponent < ViewComponent::Base
+    with_collection_parameter :field
+
+    def initialize(field:, delimiter: nil, label_html_attributes: {}, value_html_attributes: {}, value_transformer: nil)
+      super
+
+      @field = field
+      @delimiter = delimiter
+      @value_transformer = value_transformer
+      @label_html_attributes = label_html_attributes
+      @value_html_attributes = value_html_attributes
+    end
+
+    attr_reader :field, :delimiter, :label_html_attributes, :value_html_attributes
+
+    def render?
+      field.values.any?(&:present?)
+    end
+
+    def format_value(value)
+      if @value_transformer
+        @value_transformer.call(value)
+      else
+        helpers.format_mods_html(value, field: field)
+      end
+    end
+
+    def values
+      if delimiter
+        [safe_join(field.values.select(&:present?).map { |value| format_value(value) }, delimiter)]
+      else
+        field.values.select(&:present?).map { |value| format_value(value) }
+      end
+    end
+
+    def label
+      field.label&.sub(/:$/, '')
+    end
+
+    def class_for_row(index)
```

Comment:
Why is this needed?  This seems very coupled with CSS in purl, which means, it should probably go in purl and not be here.  Perhaps Purl can make a subclass of this class?

Reply:
The semantic way to do this is with multiple `<tbody>` tags.  Then you can style with `tbody tr:first-of-type`

Reply:
I don't see the problem here.  Each unique header is a new `<tbody>` with one two or three `<tr>`s in it.

Reply:
Gotcha. I hadn't thought of that. Let me see if that works with the column widths being left unspecified.

Reply:
FYI -- That worked. Removed.